### DISCUSSION TITLE
Add "Edit Entry" button to the Debug Toolbar

### DISCUSSION THREAD

**Discussion by svondervoort** [craftcms/cms#16028] [Ideas] [answered]
I would be a nice addition to add an "Edit Entry" button to the Debug Toolbar in order to go to the CP of that specific entry quickly.

**Comment by wmdhosting** [comment]
Can this help 
https://plugins.craftcms.com/palette?craft5

**Reply by svondervoort** [reply]
Pretty cool plugin, but I think it is a bit too difficult for our end-users.

**Reply by brandonkelly** [reply]
Your end users should not have admin accounts with access to the Debug Toolbar!

**Reply by svondervoort** [reply]
@brandonkelly sorry I didn't check if the Debug Toolbar was available for non-admin accounts. Another CMS we use (Kunstmaan) has a stripped down version of the Debug Toolbar for non-admin/end-users which only allows the to have an "Edit page" option.

I know from the last DotAll conference it is also possible to add the logic to the frontend using twig, but was just thinking it would be a great build-in feature.

**Reply by thupsi** [reply, marked as answer]
@svondervoort Maybe [admin bar](https://plugins.craftcms.com/admin-bar) is what you're looking for?

**Reply by svondervoort** [reply]
@thupsi this plugin also caught my eye. It is pretty customizable as well and I managed to make it look more like the default Craft 5 CP.

![image](https://github.com/user-attachments/assets/149744eb-629d-445c-acc5-0b75285aee62)
